backend/app/models/account.rb

```diff
@@ -22,6 +22,7 @@ class Account < ApplicationRecord
   validates :instagram, url: true
   validates :language, inclusion: {in: Language::TYPES}
   validates :picture, attached: true, content_type: /\Aimage\/.*\z/
+  validates :contact_email, presence: true
```

Comment:
Do we need email validation? I think devise is doing that with regex, but I'm not sure.

Reply:
`Devise.email_regexp`

Reply:
sure, added!